axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
+
+
+def load_file(filename, directory):
```

Comment:
I'm not averse to the extra dependency. :+1: 

Could change the output of the `results_set.summarize` to be a data frame too (not for this PR, another issue :)).